pkg/cli/advisory_triage.go

```diff
@@ -0,0 +1,283 @@
+func cmdAdvisoryTriage() *cobra.Command {
+	p := &triageParams{}
+	cmd := &cobra.Command{
+		Use:   "triage",
+		Short: "Triage vulnerabilities and record analysis in advisory data",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := cmd.Context()
+
+			logger := clog.NewLogger(getLogger(p.verbosity))
+			ctx = clog.WithLogger(ctx, logger)
+
+			if p.traceFile != "" {
+				w, err := os.Create(p.traceFile)
+				if err != nil {
+					return fmt.Errorf("creating trace file: %w", err)
+				}
+				defer w.Close()
+				exporter, err := stdouttrace.New(stdouttrace.WithWriter(w))
+				if err != nil {
+					return fmt.Errorf("creating stdout exporter: %w", err)
+				}
+				tp := trace.NewTracerProvider(trace.WithBatcher(exporter))
+				otel.SetTracerProvider(tp)
+
+				defer func() {
+					if err := tp.Shutdown(context.WithoutCancel(ctx)); err != nil {
+						clog.FromContext(ctx).Errorf("shutting down trace provider: %v", err)
+					}
+				}()
+
+				tctx, span := otel.Tracer("wolfictl").Start(ctx, "advisory triage")
+				defer span.End()
+				ctx = tctx
+
+				logger.Info("tracing enabled", "traceFile", p.traceFile)
+			}
+
+			// TODO: Support triaging multiple packages at once
+			if p.packageName == "" {
+				return fmt.Errorf("--%s is required", flagNamePackage)
+			}
+
+			if p.advisoriesDir == "" {
+				return fmt.Errorf("--%s is required", flagNameAdvisoriesRepoDir)
+			}
+
+			if p.builtPackagesDir == "" {
+				return fmt.Errorf("--%s is required", flagNameBuiltPackagesDir)
+			}
+
+			advIndex, err := v2.NewIndex(ctx, rwos.DirFS(p.advisoriesDir))
+			if err != nil {
+				return fmt.Errorf("creating index of advisories repo: %w", err)
+			}
+
+			if advIndex.Select().Len() == 0 {
+				return fmt.Errorf("no advisory documents found in %q", p.advisoriesDir)
+			}
+
+			o, err := target.New(ctx, os.DirFS(p.builtPackagesDir))
+			if err != nil {
+				return fmt.Errorf("constructing APK opener: %w", err)
+			}
+
+			targetAPK, err := o.LatestVersion(p.packageName)
+			if err != nil {
+				return fmt.Errorf("getting latest version of %q: %w", p.packageName, err)
+			}
+
+			apk, err := o.Open(targetAPK)
+			if err != nil {
+				return fmt.Errorf("opening APK file for %q: %w", targetAPK, err)
+			}
+			defer apk.Close()
+
+			expandedAPKTempDir, err := os.MkdirTemp("", "wolfictl-expanded-apk-*")
+			if err != nil {
+				return fmt.Errorf("creating temporary directory for expanded APK: %w", err)
+			}
+			defer os.RemoveAll(expandedAPKTempDir)
+			apkExpanded, err := expandapk.ExpandApk(ctx, apk, expandedAPKTempDir)
+			if err != nil {
+				return fmt.Errorf("expanding APK: %w", err)
+			}
+
+			// For now, the implementation peeks into the Grype database. But it'd be good
+			// to replace this with a more upstream way to get the vulnerability data, such
+			// as local OSV data stores.
+			grypeStore, err := store.New(filepath.Join(scan.GrypeDBDir(), "5", "vulnerability.db"), false)
```

Comment:
This sounds like a `TODO` here. Couldn't we use `LoadVulnerabilityDB` as done for scanning or similar.

Reply:
`LoadVulnerabilityDB` does a lot more things than we need here. It probably shouldn't, but it does for now.

This implementation boils down what we need to a minimum. It's also _hopefully_ short-lived — I'd love to get the local OSV DB thing working at some point.